TDTChocolate/FoundationAdditions/NSString+TDTAdditions.h

```diff
@@ -93,6 +93,12 @@
 - (BOOL)tdt_isEmail2;
 
 /**
+ @returns the domain of the email (string followed by the '@').
+ The method fails if the string is not an email.
```

Comment:
Yes, I agree it'd be better to return nil than an assert. But that's how this method was written in PingPong, so I didn't bother to change.